src/components/ExtensionLinks.astro

```diff
@@ -32,36 +37,21 @@ const toURL = (url: string) => {
   <li>
     <div class='container'>
       <a href={toURL(URLS.chrome)} title='Chrome Web Store'>
-        <img
-          alt='Chrome Web Store'
-          src='/img/chrome-webstore.png'
-          width='200'
-          height='200'
-        />
+        <Image src={chromeWebstore} alt='Chrome Web Store' />
```

Comment:
This png got converted to a webp, and lost some quality I think. I think we should preserve the file type in this processing.

Reply:
Having imported the images from `src` we still get the benefit of automatic width and height calculations.